state-chain/chains/src/sol/instruction_builder.rs

```diff
@@ -43,27 +36,12 @@ impl SolanaInstructionBuilder {
 		agg_key: SolPubkey,
 		destination_asset: Asset,
 		destination_address: EncodedAddress,
-		broker_id: AccountId,
-		broker_commission: BasisPoints,
-		refund_parameters: ChannelRefundParametersDecoded,
-		boost_fee: BasisPoints,
-		affiliate_fees: BoundedVec<AffiliateAndFee, ConstU32<MAX_AFFILIATES>>,
```

Comment:
I've rebased the EVM work on top of the base branch, which has Solana work merged already.

In this change I've refactored the code that builds the cf_parameter out so it can be shared across EVM and Solana
Hence this change in Solana's InstructionBuilder